packages/synapse-react-client/esbuild.config.mjs

```diff
@@ -45,6 +45,7 @@ const esBuildOptions = {
       loadPaths: ['../../node_modules', 'node_modules'],
     }),
     svgrPlugin({
+      plugins: ['@svgr/plugin-jsx'],
```

Comment:
Upgraded svgr plugins use new versions of svgr which require specifying the plugins we use. We just use the `jsx` plugin.